dropwizard/src/main/java/com/linecorp/armeria/server/dropwizard/connector/ArmeriaH2ConnectorFactory.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.dropwizard.connector;
+
+import java.security.cert.CertificateException;
+import java.util.Objects;
+
+import javax.annotation.Nullable;
+import javax.net.ssl.SSLException;
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import com.linecorp.armeria.common.Flags;
+import com.linecorp.armeria.server.ServerBuilder;
+
+import io.dropwizard.jetty.ConnectorFactory;
+
+@JsonTypeName(ArmeriaH2ConnectorFactory.TYPE)
+public class ArmeriaH2ConnectorFactory extends ArmeriaHttpsConnectorFactory {
```

Comment:
Is it necessary to have this `ConnectorFactory`? `ArmeriaHttpsConnectorFactory` already is capable of HTTP/2. Isn't it better merging this class into `ArmeriaHttpsConnectorFactory`?

Reply:
Sure. Can move the `.http2` methods over here. 

Or rename this to H2C